deepracer/edit_action_speeds/editActionSpeeds.py

```diff
@@ -0,0 +1,61 @@
+import json
+import os
+import uuid
+
+
+# README
+# Paste in your full action space into the file "action_space.json", it can handle index values if you have them already
+# Run this script with desired parameters on the last line"
+
+
+def edit_action_speeds(filename, edit_value, change_left_steering, change_straight, change_right_steering, add_indexer):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using more than five non-keyword parameters in a function definition can indicate that the function is difficult to understand and maintain. Try to combine parameters in objects or refactor the function into small functions.

Reply:
Agreed